src/page-parole/PageParole.js

```diff
@@ -1,5 +1,11 @@
 import React from "react";
 import DetailPage from "../detail-page";
+// once the backend is in place, stop using these untracked test files
```

Comment:
@macfarlandian, when you share your care package with @sprad, make sure to send him these files, as well. Thanks!